src/base/bittorrent/session.cpp

```diff
@@ -1496,6 +1434,83 @@ void Session::loadLTSettings(lt::settings_pack &settingsPack)
     }
 }
 
+void Session::configureNetInterfaces(lt::settings_pack &settingsPack)
+{
+    if (!m_listenInterfaceConfigured) {
+        const int port = useRandomPort() ? 0 : this->port();
+        if (port > 0)  // user specified port
+            settingsPack.set_int(lt::settings_pack::max_retry_port_bind, 0);
+
+        QStringList endpoints;
+        QStringList outgoingInterfaces;
+        const QStringList listeningIPs = getListeningIPs();
+        const QString portString = ':' + QString::number(port);
+
+        //const QString firstIP = listeningIPs[0];
+        if (listeningIPs.isEmpty()) {
+            endpoints << QHostAddress(QHostAddress::AnyIPv4).toString() + portString;
```

Comment:
Maybe define some helper like:
```c++
QString makeEndpointString(QHostAddress ip, int port);
```

Reply:
Nah, I don't think this is worth it. It is only 2-3 places where we make endpoints.

Reply:
Just to clarify: I don't see a point to this so I am not going to implement it.

Reply:
Ok. Don't insist.